.storybook/main.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+/* eslint-disable require-unicode-regexp */
```

Comment:
why disabled rather than adding `/u` to regexes? Does that make webpack unhappy?

Reply:
Correct. Only in some instances as well which is the most confusing part.